easyrest/models/user.py

```diff
@@ -64,9 +74,12 @@ def add(database, form_data):
             "required": ["name", "email", "password"]
         }
         validation(schema, form_data)
-        name = form_data['name']
         email = form_data['email']
+        email_in_database = database.query(User.email).filter(User.email == email).scalar()
+        if email_in_database is not None:
+            raise ValidationError('already exist', email)
```

Comment:
Maybe something like DuplicationError better suited here? Just to not overuse ValidationError.

Reply:
My opinion - don't complicate things. To make a duplication error, you need to define it. Then, according to the code above in the `user_controller.py` (`try` section, line 165 and 180), you already have to expect 2 exceptions, and this is despite the fact that the current exception (`ValidationError`) fits the context.